app/views/devise/confirmations/new.html.slim

```diff
@@ -0,0 +1,14 @@
+= bootstrap_devise_error_messages!
+.row
+  .col-md-6.offset-md-3
+    .panel.panel-default.devise-bs
+      .panel-heading.text-center
+        h4
+          = t('.resend_confirmation_instructions', default: 'Resend confirmation instructions')
```

Comment:
@sinpai no need in `:default` option, let's just keep locales up to date.

In order to add locale [check this article](http://guides.rubyonrails.org/i18n.html)

I've noticed it in several places more. So, be sure to remove this `default` option keys everywhere and add needed locales to `en.yml`

Reply:
:+1: